group_vars/htcondor/vars.yml

```diff
@@ -1,11 +1,12 @@
 # Configure nodes in the HTCondor cluster.
 ---
-htcondor_server: "condor-cm.galaxyproject.eu"
```

Comment:
should we update it also here then, if we want to name it by `build`?
https://github.com/usegalaxy-eu/vgcn-infrastructure/blob/e2cb8a4e5fb74bbcf210d175d15f40f321928a94/userdata.yaml.j2#L7

Reply:
In your [PR](https://github.com/usegalaxy-eu/vgcn-infrastructure/pull/283/files#diff-18205f92094934ee2fc32afb35cfc40957322c9f197d3707a753bc02484bd5ba) you have already removed it, right?

Reply:
yes, it is "baked-in" here:
https://github.com/usegalaxy-eu/vgcn/blob/2a80ce37ca3f1179c16bdfc026e7141dfb71fe9e/ansible/group_vars/condor.yml#L2

Reply:
Yup, it's baked into the image itself.